Library/Styles/SettingsStyles.swift

```diff
@@ -79,3 +79,53 @@ public func settingsAttributedPlaceholder(_ string: String) -> NSAttributedStrin
     attributes: [NSAttributedString.Key.foregroundColor: UIColor.ksr_text_dark_grey_400]
   )
 }
+
+private let settingsLayoutMarginsStyle =
+  UIView.lens.layoutMargins .~ UIEdgeInsets(topBottom: Styles.grid(2), leftRight: Styles.grid(1))
+    <> UIView.lens.preservesSuperviewLayoutMargins .~ false
+
+public let settingsContentViewStyle = settingsLayoutMarginsStyle
+
+private let settingsHeaderLayoutMargins = UIEdgeInsets(
+  top: Styles.grid(5),
+  left: Styles.grid(1),
+  bottom: Styles.grid(2),
+  right: Styles.grid(1)
+)
+
+public let settingsHeaderContentViewStyle =
+  settingsLayoutMarginsStyle
+    <> UIView.lens.layoutMargins .~ settingsHeaderLayoutMargins
+
+public let settingsFooterContentViewStyle =
+  settingsLayoutMarginsStyle
+
+public let settingsHeaderFooterLabelBaseStyle =
+  UILabel.lens.font %~ { _ in .ksr_footnote() }
```

Comment:
Notice that the `font` needs to use `%~` this is due to the fact that we don't want it to cache the old size value. We have paired on this with Justin and is one of the major issues why fonts are not resizing properly in Settings. I'll be fixing those separately.